android/src/main/java/io/wazo/callkeep/RNCallKeepModule.java

```diff
@@ -196,6 +215,7 @@ public void startCall(String uuid, String number, String callerName) {
         Log.d(TAG, "startCall called, uuid: " + uuid + ", number: " + number + ", callerName: " + callerName);
 
         if (!isConnectionServiceAvailable() || !hasPhoneAccount() || !hasPermissions() || number == null) {
+            Log.d(TAG, "startCall ignored: " + isConnectionServiceAvailable() + ", " + hasPhoneAccount() + ", " + hasPermissions() + ", " + number);
```

Comment:
I've added this because I've been screwed over by this silent return more than once. I hope we can add this, but it's not strictly related to self managed.